File: 0977_Squares_of_a_Sorted_Array/977.py
class Solution:
    def sortedSquares(self, nums: list[int]) -> list[int]:
        return sorted([i * i for i in nums])

a = Solution()
print(a.sortedSquares(nums = [-4,-1,0,3,10]))
print(a.sortedSquares(nums = [-7,-3,2,3,11]))

File: 0977_Squares_of_a_Sorted_Array/977.go
package main

import (
	"fmt"
	"sort"
)

func sortedSquares(nums []int) []int {
	var ans []int
	for _, i := range nums {
		ans = append(ans, i*i)
	}
	// for _, i := range ans {
	// 	fmt.Println(i)
	// }
	sort.Ints(ans)
	return ans
}

func main() {
	nums := [5]int{-4, -1, 0, 3, 10}
	for _, i := range sortedSquares(nums[:]) {
		fmt.Println(i)
	}
}
